File: Java/Leccion01/MundoPc/src/ar/com/system2023/mundopc/Orden.java

package ar.com.system2023.mundopc;


public class Orden {
    private final int idOrden;
    private Computadora computadora[];// arreglo de objetos 
    private static int contadorOrden;
    private static final int MAX_COMPUTADORAS = 10;
    private int contadorComputadora;
    
    //constructor vacio
    public Orden(){
        this.idOrden = ++Orden.contadorOrden;
        this.computadora = new Computadora[Orden.MAX_COMPUTADORAS];
    }
    
    //metodo para agregar una nueva computadora al arreglo
    public void agregarComputadora(Computadora computadora){
        if(this.contadorComputadora < Orden.MAX_COMPUTADORAS){
            this.computadora[this.contadorComputadora++] = computadora;
        }
        else{
            System.out.println("Has Superado el limite: "+Orden.MAX_COMPUTADORAS);
        }
    }
    
    
    
}

File: Python/Leccion14/mundo_pc/orden.py
class Orden:

    contador_ordenes = 0

    def _init_(self, computadoras):
        Orden.contador_ordenes += 1
        self._id_orden = Orden.contador_ordenes
        self._computadoras = computadoras

    # Se crea la lista
    def agregar_computadora(self, computadoras):
        self._computadoras.append(computadoras)

    def _str_(self):
        computadoras_str = ''
        for computadora in self._computadoras:
            computadoras_str += computadora._str_()
        return f'''
            Orden: {self._id_orden}
            Computadora: {computadoras_str}
        '''